src/requirements/data/checkers-common.ts

```diff
@@ -39,7 +38,6 @@ export const courseMatchesCodeOptions = (course: Course, codeOptions: readonly s
  * @returns if the course satisfies FWS requirement.
  */
 export const courseIsFWS = (course: Course): boolean =>
-  course.crseId === FWS_COURSE_ID ||
```

Comment:
I removed this because during the checker/precomputation phase, a `crseId` will never be our custom `FWS_COURSE_ID`. Adding courses with our custom course ids (type `CourseTaken`, opposed to `Course`) happens after the AP/IB exam conversion, during the frontend computation.